probe/endpoint/ebpf.go

```diff
@@ -0,0 +1,285 @@
+package endpoint
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+	"syscall"
+	"unsafe"
+
+	log "github.com/Sirupsen/logrus"
+	bpflib "github.com/kinvolk/gobpf-elf-loader/bpf"
+)
+
+import "C"
+
+var byteOrder binary.ByteOrder
+
+func init() {
+	var i int32 = 0x01020304
+	u := unsafe.Pointer(&i)
+	pb := (*byte)(u)
+	b := *pb
+	if b == 0x04 {
+		byteOrder = binary.LittleEndian
+	} else {
+		byteOrder = binary.BigEndian
+	}
+}
+
+type EventType uint32
```

Comment:
this does not build in CircleCI: https://circleci.com/gh/alepuccetti/scope/51

you should unexport it s/EventType/eventType/g